deepmd/pt/model/atomic_model/linear_atomic_model.py

```diff
@@ -188,36 +190,46 @@ def forward_atomic(
 
         for i, model in enumerate(self.models):
             mapping = self.mapping_list[i]
+            raw_ret = model.forward_atomic(
```

Comment:
`forward_common_atomic` should be called. You do not need to take care the `apply_out_stat` for each individual model. 
note that the mapped atype is used in `forward_common_atomic`.